src/analysis/retail/shaman/restoration/modules/talents/FlashFlood.tsx

```diff
@@ -63,6 +63,8 @@ class FlashFlood extends Analyzer {
   constructor(options: Options) {
     super(options);
     this.active = this.selectedCombatant.hasTalent(TALENTS.FLASH_FLOOD_TALENT.id);
```

Comment:
I'd recommend adding a return if the module should not be active before adding listeners and doing other logic.

Reply:
I've been specifically suggesting not to do this. It's hidden behavior, but listeners actually don't get added for inactive modules.

Reply:
Oh interesting. I'm aware that listeners are not executed. The reason I've been suggesting this isbecause there can be unused code still running. Like calculating the effectiveness of an ability. Eventho its not a listener it is still code that has to be exectued in the browser - aka the end-user.

Reply:
It's such a trivial amount of extra code I doubt it matters much, but point taken. I still think the no return looks cleaner 🤷